Formula/rsyslog.rb

```diff
@@ -1,13 +1,13 @@
 class Rsyslog < Formula
   desc "Enhanced, multi-threaded syslogd"
   homepage "https://www.rsyslog.com/"
-  url "https://www.rsyslog.com/files/download/rsyslog/rsyslog-8.2104.0.tar.gz"
-  sha256 "710981c3c34f88d5d1fb55ecfc042aecad8af69414b2b1602b304f4dedbf9f43"
+  url "https://github.com/rsyslog/rsyslog/archive/refs/tags/v8.2106.0.tar.gz"
+  sha256 "9f3e2113affe5ac28e181ff647631830461ccc6c45bd3a69d61fd7d2d023e360"
   license all_of: ["Apache-2.0", "GPL-3.0-or-later", "LGPL-3.0-or-later"]
 
   livecheck do
-    url :homepage
-    regex(/Current Version.+?v?(\d+(?:\.\d+)+)/im)
+    url :stable
+    strategy :github_latest
```

Comment:
If we are planning to go ahead with the GitHub stable URL, we can use the standard regex for tags:
```suggestion
    url :stable
    regex(/^v?(\d+(?:\.\d+)+)$/)
```
However, I'm not sure whether we should move to the GitHub source, even though the homepage is seemingly outdated (seeing as this version should've come out in June 2021 according to the homepage). This is especially because of the addition of build dependencies, and maybe we should ask upstream if this version has been released (with the configure script in the archive).

Reply:
[This post](https://www.adiscon.com/news/rsyslog-8-2106-0-released/) says version 8.2106.0 is released.

Reply:
Thanks for the link. Perhaps we could use the this tarball instead: https://www.rsyslog.com/files/download/rsyslog/rsyslog-8.2106.0.tar.gz, since it comes with the configure script and we can avoid the additional build dependencies (probably).

As for the `livecheck` block, we could check the location of the stable archive:
```suggestion
    url "https://www.rsyslog.com/downloads/download-v8-stable/"
    regex(/href=.*?rsyslog[._-]v?(\d+(?:\.\d+)+)\.t/i)
```

Reply:
Didn't see your comment; updated to use https://www.rsyslog.com/news-releases/. Not sure which is preferable.

Reply:
We prefer matching version from URLs rather than text, as they are less likely to undergo changes in format, and text is often unreliable. We aim to match URLs at the location of the stable archive, as that would be useful in retrieving the latest URL (and in the future could be used to make automated PRs).